src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -266,18 +267,20 @@ void reportWavefrontGeneratedData(WavefrontSpan span) {
       put(COMPONENT_TAG_KEY, span.getComponentTagValue());
     }};
 
-    boolean customTagMatch = false;
     // avoid iterating through the span tags if user has not instantiated any red-metric custom tags
-    if (redMetricsCustomTagKeys.size() > 0) {
+    if (redMetricsCustomTagKeys.size() > 1) {
```

Comment:
why is this > 1?

Reply:
Change the way of writing this logic, hope it's clearer now.